api/base/utils.py

```diff
@@ -137,3 +137,9 @@ def default_node_permission_query(user):
 
 def extend_querystring_params(url, params):
     return furl.furl(url).add(args=params).url
+
+def extract_object_from_dict(self, obj):
```

Comment:
Added in https://github.com/CenterForOpenScience/osf.io/pull/5532/commits/3583141ebb4975a2e1ebe54b7e83b0ee2201e50c